helm/aws-load-balancer-controller/values.yaml

```diff
@@ -140,9 +140,15 @@ vpcId:
 # Custom AWS API Endpoints (serviceID1=URL1,serviceID2=URL2)
 awsApiEndpoints:
 
+# Custom AWS API throttle settings (serviceID1:operationRegex1=rate:burst,serviceID2:operationRegex2=rate:burst)
```

Comment:
Lets specify the comments in this format:
\# awsApiThrottle specifies custom AWS API ...

Reply:
Also, could we have some examples in the comments?

Reply:
make sense